amplfi/train/architectures/embeddings/multimodal.py

```diff
@@ -0,0 +1,67 @@
+from typing import Literal, Optional
+
+import torch
+from ml4gw.nn.norm import NormLayer
+from ml4gw.nn.resnet.resnet_1d import ResNet1D
+
+from amplfi.architectures.embeddings.base import Embedding
```

Comment:
I think this needs to be `amplfi.train.architectures.embeddings.base`

Reply:
that's correct, there is a typo. But let's use relative imports anyway, they are good practice in a single repo and any circular initialization is caught rightaway.

Reply:
I changed it in the `multimodal.py`, and created a new PR #148 for adding relative imports to all the other files.